monitoring/monit.py

```diff
@@ -86,8 +88,35 @@ def run_command(command):
         module.run_command('%s %s %s' % (MONIT, command, name), check_rc=True)
         return status()
 
-    process_status = status()
-    present = process_status != ''
+    def wait_for_monit_to_stop_pending(sleep_time=1):
```

Comment:
I'd say keep it simple and go with say a straight 5s.

We need to balance wasting CPU and slowing things down by polling too much, versus increased latency.